user/enterprise/installation.md

```diff
@@ -30,14 +30,15 @@ If you're setting up your AMI for the first time you need to create
 a Security Group. From the EC2 management console, create an entry for
 each port in the table below:
 
-| Port | Service         | Description                                                                  |
-|:-----|:----------------|:-----------------------------------------------------------------------------|
-| 8800 | Custom TCP Rule | This port is to access the admin dashboard for your Enterprise installation. |
-| 5672 | Custom TCP Rule | For RabbitMQ Non-SSL.                                                        |
-| 4567 | Custom TCP Rule | For RabbitMQ SSL.                                                            |
-| 443  | HTTPS           | Web application over HTTPS access.                                           |
-| 80   | HTTP            | Web application access.                                                      |
-| 22   | SSH             | SSH access.                                                                  |
+| Port | Source / Origin           | Service         | Description                                                                  |
+|:-----|:--------------------------|:----------------|:-----------------------------------------------------------------------------|
+| 8800 | Administrators' computers | Custom TCP Rule | This port is to access the admin dashboard for your Enterprise installation. |
```

Comment:
why do we describe this port differently? Can you just say Enterprise Admin Dashboard? And probably remove "for" in the RabbitMQ lines as well